ansible_collections/arista/avd/docs/plugins/Filter plugins/convert_dicts.md

```diff
@@ -0,0 +1,76 @@
+# convert_dicts
+
+!!! note
+    Always use the FQCN (Fully Qualified Collection Name) `arista.avd.convert_dicts` when using this plugin.
+
+Convert a dictionary containing nested dictionaries to a list of dictionaries\.
+
+## Synopsis
+
+The filter inserts the outer dictionary keys into each list item using the primary\_key \`name\` \(the key name is configurable\)\, and if there is a non\-dictionary value\, it inserts this value to secondary key \(the key name is configurable\)\, if <em>secondary\_key</em> is provided\.
```

Comment:
Some of the `\` are present in the [preview site](https://ansible-avd--3061.org.readthedocs.build/en/3061/docs/plugins/Filter%20plugins/convert_dicts.html#synopsis)